app/api/score_sub.py

```diff
@@ -354,6 +354,11 @@ async def submit_score(
 
     await app.usecases.stats.save(stats)
 
+    await app.usecases.beatmap.increment_playcount(
+        beatmap=beatmap,
+        increment_passcount=score.passed,
+    )
```

Comment:
Seems like there are no cases `return`ing above where this would be desirable

Reply:
Maybe this one is debatable:

https://github.com/osuAkatsuki/score-service/blob/9c0d449fbaeebc4b83745662d8b67badd70ae8cf/app/api/score_sub.py#L151-L152